### PR TITLE
[wallet-ext] Manage Accounts UI 

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Added some basic UI for the "Manage Accounts" page as well as tied together some of the other elements from previous iterations. 

There's likely some clean-up / consolidation we can do but wanted to get the PR up to keep moving forward. 

https://github.com/MystenLabs/sui/assets/122397493/5d7501a4-9ac8-4c47-b246-ae265048ed83

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/ui/app/components/accounts/UnlockAccountModal.tsx
Comment (mamos-mysten) [inline apps/wallet/src/ui/app/components/accounts/UnlockAccountModal.tsx:46]: @williamrobertson13 @pchrysochoidis this might not be necessary at this point, but I updated this to match the `PasswordInputDialog` component. need to double check the designs but we should move forward with this one

File: apps/wallet/src/ui/app/shared/forms/controls/PasswordInput.tsx
Comment (mamos-mysten) [inline apps/wallet/src/ui/app/shared/forms/controls/PasswordInput.tsx:27]: this was added so the Tab key skips over the show / hide icon. not 100% sure if there's a better way to handle this so that it's still accessible

File: apps/wallet/src/ui/app/pages/accounts/manage/ManageAccountsPage.tsx
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/pages/accounts/manage/ManageAccountsPage.tsx:54]: lets move this and the other components to their own file. We should probably rename this to something like `AccountsGroup` or something like that. We can have multiple account groups with the same type. Like 2 zk with different provider or even same provider but different email.

We might need to pass title as well but we can update that when necessary. Also maybe specific props based on the account type, like mnemonic source id if accounts are mnemonic-derived.

**Review by pchrysochoidis — APPROVED** [review]
I added a few comments - I will approve to unblock but we should address some of them before merging (like moving components to their own files and changing the order of routes)

File: apps/wallet/src/ui/app/shared/forms/controls/PasswordInput.tsx
Comment (mamos-mysten) [reply apps/wallet/src/ui/app/shared/forms/controls/PasswordInput.tsx:27]: so you can easily tab between enter / confirm password fields

File: apps/wallet/package.json
Comment (mamos-mysten) [reply apps/wallet/package.json:21]: accident

File: apps/wallet/tests/utils/localnet.ts
Comment (mamos-mysten) [reply apps/wallet/tests/utils/localnet.ts:4]: was trying to get playwright running in VSCode, but ultimately abandoned it for the CLI. accidentally left these changes in here!

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx
Comment (mamos-mysten) [reply apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:53]: agreed, was just stayling in line with the current state of TokensDetails 🥲

**Pull request merged**
